docs/_docs/contributing/tools/index.md

```diff
@@ -2,3 +2,8 @@
 layout: index
 title: IDEs and Tools
 ---
+
+This chapter of the guide describes how to use Dotty with IDEs and other tools:
+- [IDEs](./ides.md)
+- [Use Mill](mill.md)
```

Comment:
It seems those files are missing, I only see `scalafix.md`

Reply:
Ho yes, a little hint sorry